### DISCUSSION TITLE
SessionEnv was unavailable to handle a notification event

### DISCUSSION THREAD

**Discussion by 5MI7th3MI6** [Security-Onion-Solutions/securityonion#9076] [Unsupported Versions] [answered]
From the Security Onion course Developing Your Detection Playbook with Security Onion 2, I am testing out the rule (seen in the image):
![image](https://user-images.githubusercontent.com/68974383/200155314-08498eb5-d07b-42d9-858f-5c6788087d1e.png)
However, I get this error, SessionEnv was unavailable to handle a notification event,  in the alerts section when I trigger the rule.

**Comment by defensivedepth** [comment, marked as answer]
What Event ID is the log that generated the alert?

Also, since this is part of the online training, I went ahead and reached out to you via the training platform.
